src/Sql/Sql/help/Get-AzSqlServerTrustGroup.md

```diff
@@ -0,0 +1,177 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/powershell/module/az.sql/get-azsqlservertrustgroup
+schema: 2.0.0
+---
+
+# Get-AzSqlServerTrustGroup
+
+## SYNOPSIS
+Gets information about Server Trust Group.
+
+## SYNTAX
+
+### GetByName
+```
+Get-AzSqlServerTrustGroup [-ResourceGroupName] <String> [-Location] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ListByInstanceSet
+```
+Get-AzSqlServerTrustGroup [-ResourceGroupName] <String> [-InstanceName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ListByLocationSet
+```
+Get-AzSqlServerTrustGroup [-ResourceGroupName] <String> [-Location] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdSet
+```
+Get-AzSqlServerTrustGroup [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzSqlServerTrustGroup** cmdlet get information about Server Trust Group and its members for current subscription. Based on parameter set this cmdlet can retrieve specified Server Trust Group, all Server Trust Groups in a specified location or Server Trust Groups that have specified AzureSQL Managed Instance as a member.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzSqlServerTrustGroup -Location "West Europe" -Name "ServerTrustGroup01" -ResourceGroupName "ResourceGroup01"
+```
+
+Gets information about Server Trust Group named ServerTrustGroup in resource group ResourceGroup in location West Europe.
```

Comment:
ServerTrustGroup01 instead of ServerTrustGroup
ResourceGroup01 instead of ResourceGroup